webpack.config.js

```diff
@@ -78,5 +78,44 @@ module.exports = [
     resolve: {
       extensions: ['.js', '.jsx', '.ts', '.tsx'],
     }
+  },
+  {
+    entry: './src/main/frontend/multi-pipeline-graph-view',
```

Comment:
there's probably a simpler way than duplicating all this config each time,
e.g. see Jenkins core: https://github.com/jenkinsci/jenkins/blob/master/war/webpack.config.js

not a blocker